contracts/interchain-token-service/src/contract/execute/mod.rs

```diff
@@ -163,18 +168,39 @@ fn apply_to_hub(
 
 fn apply_to_transfer(
     storage: &mut dyn Storage,
+    querier: QuerierWrapper,
     source_chain: ChainNameRaw,
     destination_chain: ChainNameRaw,
     transfer: InterchainTransfer,
 ) -> Result<InterchainTransfer, Error> {
-    interceptors::subtract_supply_amount(storage, &source_chain, &transfer)?;
-    let transfer = interceptors::apply_scaling_factor_to_amount(
-        storage,
-        &source_chain,
-        &destination_chain,
-        transfer,
-    )?;
-    interceptors::add_supply_amount(storage, &destination_chain, &transfer)?;
+    // we only do balance tracking for amplifier chains
+    let client: nexus::Client = client::CosmosClient::new(querier).into();
+    let source_is_amplifier_chain = !client
+        .is_chain_registered(&source_chain.normalize())
+        .change_context(Error::Nexus(source_chain.clone()))?;
+    let destination_is_amplifier_chain = !client
+        .is_chain_registered(&destination_chain.normalize())
+        .change_context(Error::Nexus(destination_chain.clone()))?;
+
+    if source_is_amplifier_chain {
+        interceptors::subtract_supply_amount(storage, &source_chain, &transfer)?;
+    }
+
+    // we only possibly scale if both source chain and destination chain are amplifier chains
```

Comment:
We still need to scale for consensus chains. E.g. Ethereum <-> Sui

Instead of skipping the interceptors completely for consensus chains, we need to apply the interceptors, but if the token instance can't be loaded and the chain is consensus, then we will need to use the "default" instance instead of failing. Perhaps the default instance can be the Untracked supply and the decimals from the token instance for the origin chain.

This is more convoluted though. We could consider exposing a setter for the token instance for consensus chain and just migrate a few popular tokens but the others would fail